activesupport/lib/active_support/core_ext/securerandom.rb

```diff
@@ -22,4 +24,24 @@ def self.base58(n = 16)
       BASE58_ALPHABET[idx]
     end.join
   end
+
+  # SecureRandom.base36 generates a random base36 string in lowercase.
+  #
+  # The argument _n_ specifies the length, of the random string to be generated.
```

Comment:
I know this came from the `base58` docs, but the comma after “length” is unnecessary. Can you remove it here and above?